DistributionPackages/Neos.NeosIo/Resources/Private/Fusion/NodeTypes/CountUp.fusion

```diff
@@ -0,0 +1,20 @@
+prototype(Neos.NeosIo:CountUp) < prototype(Neos.Neos:ContentComponent) {
+    rawNumber = ${String.stripTags(String.trim(q(node).property('number')))}
+                                                
+    renderer = afx`
+        <div class="countUp u-alignCenter">
+            <div class="countUp__numberLine h1 u-mb0">
+                <Neos.Neos:Editable property="prefix" block={false} />
+                <span data-component="CountUp" data-component.@if.live={!node.context.inBackend} data-to={props.rawNumber}>
+                    <Neos.Neos:Editable property="number" block={false} />
+                </span>
+                <Neos.Neos:Editable property="suffix" block={false} />
+            </div>
+            <div class="h4">
+                <span>
```

Comment:
Should the `<span>` not be removed? In the Fluid the `<span>` is only in the backend view

Reply:
Yes I wanted to, but first wanted to test and then forgot...